PKGBUILD.MSYS2

```diff
@@ -0,0 +1,64 @@
+# Themis build script for pacman in MSYS2 environment
+#
+# Maintainer: Cossack Labs Limited <dev@cossacklabs.com>
+
+pkgname=('themis' 'themis-devel')
+pkgbase=themis
+pkgver=0.11.2
+pkgrel=1
+
+pkgdesc="Data security library for network communication and data storage"
+url="https://www.cossacklabs.com/themis/"
+license=('Apache')
+changelog=CHANGELOG.md
+
+arch=('x86_64' 'i686')
+depends=('libopenssl>=1.1.1')
+makedepends=('tar' 'gcc' 'make' 'openssl-devel>=1.1.1')
+
+source=("https://github.com/cossacklabs/themis/archive/$pkgver.tar.gz")
+sha256sums=('ee5c4be360401094dc5429035e1499912b3ccc10e93f71d0c2578934a5b2c81f')
+sha1sums=('418d71b0d3dd1ffff93ef025908bacb99d4c4aa4')
+md5sums=('afc3ac4e7d6c6db59e4e2f059a4fc973')
+# TODO: verify package signature
```

Comment:
@shadinua what's the best way to verify signatures?

Reply:
[See here](https://wiki.archlinux.org/index.php/Makepkg#Signature_checking) how **makepkg** verifies signatures. Basically, we need to ensure that our PGP key is installed into the system and written down in `validpgpkeys` variable here in PKGBUILD file. Then we add the signature file to the `sources` array.

Alternatively, I imagine that we could use Git source to download the source code from GitHub. Then I guess we will need to use PGP-signed tags for releases and makepkg will verify that.

Note that this is verification of _the source code_ for building the package. It's separate from signing the resulting binary packages. Package signing is documented [here](https://wiki.archlinux.org/index.php/DeveloperWiki:Package_signing).